<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Single" duration-ms="60183" started-at="2017-03-17T13:12:06Z" finished-at="2017-03-17T13:13:06Z">
    <groups>
    </groups>
    <test name="TestSuite" duration-ms="60183" started-at="2017-03-17T13:12:06Z" finished-at="2017-03-17T13:13:06Z">
      <class name="stagingURL.TC30_Send_Location_details_to_email_address">
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:stagingURL.TC30_Send_Location_details_to_email_address@77cd7a0]" name="setUp" is-config="true" duration-ms="2359" started-at="2017-03-17T18:42:06Z" finished-at="2017-03-17T18:42:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[single.webdriver.conf.json]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="handleTestMethodName(java.lang.reflect.Method)[pri:0, instance:stagingURL.TC30_Send_Location_details_to_email_address@77cd7a0]" name="handleTestMethodName" is-config="true" duration-ms="1" started-at="2017-03-17T18:42:09Z" finished-at="2017-03-17T18:42:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void stagingURL.TC30_Send_Location_details_to_email_address.send_location_details_to_email_address() throws java.lang.InterruptedException,java.io.IOException,java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- handleTestMethodName -->
        <test-method status="FAIL" signature="send_location_details_to_email_address()[pri:0, instance:stagingURL.TC30_Send_Location_details_to_email_address@77cd7a0]" name="send_location_details_to_email_address" duration-ms="56771" started-at="2017-03-17T18:42:09Z" finished-at="2017-03-17T18:43:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	expected [Your results have been sent to your e-mail address.

Thanks!] but found [Your results have been sent to your e-mail address.

 Thanks!],
	expected [Your results have been sent to your e-mail address.

Thanks!] but found [Your results have been sent to your e-mail address.

 Thanks!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Your results have been sent to your e-mail address.

Thanks!] but found [Your results have been sent to your e-mail address.

 Thanks!],
	expected [Your results have been sent to your e-mail address.

Thanks!] but found [Your results have been sent to your e-mail address.

 Thanks!]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at stagingURL.TC30_Send_Location_details_to_email_address.send_location_details_to_email_address(TC30_Send_Location_details_to_email_address.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- send_location_details_to_email_address -->
        <test-method status="PASS" signature="getResult()[pri:0, instance:stagingURL.TC30_Send_Location_details_to_email_address@77cd7a0]" name="getResult" is-config="true" duration-ms="762" started-at="2017-03-17T18:43:05Z" finished-at="2017-03-17T18:43:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:stagingURL.TC30_Send_Location_details_to_email_address@77cd7a0]" name="tearDown" is-config="true" duration-ms="255" started-at="2017-03-17T18:43:06Z" finished-at="2017-03-17T18:43:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- stagingURL.TC30_Send_Location_details_to_email_address -->
    </test> <!-- TestSuite -->
  </suite> <!-- Single -->
</testng-results>
